### ISSUE TITLE
Discrepancy between Conover's test on Friedman-type ranked data in old PMCMR package and PMCMRplus

### ISSUE THREAD

**Issue by ldk-linux** [cran/PMCMRplus#1]
Running posthoc.friedman.conover.test from PMCMR and frdAllPairsConoverTest from PMCMRplus return very different statistic and p-value, even though to the best of my understanding these tests should be for all purpose equivalent. This can be observed on any test input, including the one in the vignette.

At a glance, the PMCMR output seems correct, while the PMCMRplus output seems not. However, I may be misinterpreting what frdAllPairsConoverTest is supposed to return. If so, feel free to ignore this report.

**Comment by gaborcsardi** [comment]
Hi, this is a read only mirror of CRAN, please see the package authors in the `DESCRIPTION` file. Look for `Maintainer`, `BugReports` and `URL`. Thanks!

**Comment by ldk-linux** [comment]
Yep, I realized that shortly after posting this :-) I already contacted the author, feel free to close this!
